README.md

```diff
@@ -1 +1,12 @@
-# AQA_Training
\ No newline at end of file
+# AQA_Training
+
+Required
+
+- node
+- playwright
+
+Commands
+
+run all - npx playwright test
```

Comment:
Please add these script in https://github.com/HeorhiiH/AQA_Training/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R6